packages/flutter/test/widgets/basic_test.dart

```diff
@@ -145,6 +145,71 @@ void main() {
       await tester.tap(find.byKey(key1));
       expect(_pointerDown, isTrue);
     });
+
+    testWidgets('semantics bounds are updated', (WidgetTester tester) async {
+      final GlobalKey fractionalTranslationKey = GlobalKey();
+      final GlobalKey textKey = GlobalKey();
+      Offset offset = const Offset(0.4, 0.4);
+
+      await tester.pumpWidget(
+        StatefulBuilder(
+          builder: (BuildContext context, StateSetter setState) {
+            return Directionality(
+              textDirection: TextDirection.ltr,
+              child: Center(
+                child: Semantics(
+                  explicitChildNodes: true,
+                  child: FractionalTranslation(
+                    key: fractionalTranslationKey,
+                    translation: offset,
+                    transformHitTests: true,
+                    child: Listener(
```

Comment:
Use a regular GestureDetector.onTap here instead of the low-level Listener?

Reply:
Done